spotbugsTestCases/src/java/FindReturnRefNegativeTest.java

```diff
@@ -0,0 +1,207 @@
+import edu.umd.cs.findbugs.annotations.NoWarning;
+
+import java.util.Date;
+import java.util.Hashtable;
```

Comment:
Why not `Map`?

Reply:
I copied the original test from the SEI CERT rule, but now I changed it to HashMap.